project/ValidatePullRequest.scala

```diff
@@ -360,9 +359,12 @@ object AggregatePRValidation extends AutoPlugin {
         case KeyValue(key, i: Inc) => KeyValue(key, i)
       }
 
+      /*
```

Comment:
Maybe we can keep this file as is and just remove the versions to check against in `project/Mima.scala`?

Reply:
I.e. make `mimaPreviousArtifacts` empty.

Reply:
done